lib/actions/populate.js

```diff
@@ -27,7 +27,7 @@ module.exports = function expand(route, options) {
         const Relation = actionUtil.modelFromParam(options.associationAttr);
 
         if (!Relation || !Model) {
-            return reply(Boom.notFound());
+            return Boom.notFound('Invalid model or relation');
```

Comment:
I almost wonder if this should be an internal error.  Does it point to misconfiguration?

Reply:
This makes sense.  I was thinking of it in terms of `/model/relation` doesn't exist, so 404, but we'd never get here if a route matching `/model/relation` hadn't been set up with a Tandy handler.  Will switch.